src/Components/Products/index.jsx

```diff
@@ -30,7 +33,7 @@ function Products() {
                                         </Typography>
                                     </CardContent>
                                     <CardActions>
-                                        <Button size="small">ADD TO CART</Button>
+                                        <Button onClick={() => dispatch(add(product))} size="small">ADD TO CART</Button>
                                         <Button size="small">VIEW DETAILS</Button>
                                     </CardActions>
```

Comment:
We either want to hide the card or disable the add to cart button when we don't have any of that item left in stock.